data/Templates/FhirToHL7v2/Mutators/_PatientIdentifiersToCX.liquid

```diff
@@ -0,0 +1,70 @@
+{%comment%}
+{%endcomment%}
+{%- assign acceptedCodes = "MR, SS" | split: ", " -%}
+{%- capture acceptedSystemsStr -%}
+http://example.com/v2-to-fhir-converter/assigning-authority-universal-system-HCD,
+http://example.com/v2-to-fhir-converter/assigning-authority-local-system-USSSA,
+{%- endcapture -%}
+{%- assign acceptedSystems = acceptedSystemsStr | strip_newlines | split: "," -%}
+{%-comment-%} skip the first item that is sourced from PID-2 {%-endcomment-%}
+{%- assign filteredIds = identifiers | slice_array: 1 -%}
+{%- assign filteredIds = filteredIds | select_or: "type.coding[0].code", acceptedCodes -%}
```

Comment:
Do you want `coding[0]` here or `coding[]`? For me it's usually been the nonspecific index.

Reply:
For all of the identifiers I've found there is only one coding value and I think in my reading of the fhir mapping that I just want the first one.